code/game/objects/items/stacks/medical.dm

```diff
@@ -273,6 +287,58 @@
 			else
 				to_chat(user, "<span class='notice'>The [affecting.name] is cut open, you'll need more than a bandage!</span>")
 
+// Space Klot. This stops bleeding and has a small chance to stop some internal bleeding, but it will be locked behind a tech tree + does not disinfect like a normal bruise pack.
+/obj/item/stack/medical/advanced/bruise_pack/spaceklot
+	name = "space klot"
+	singular_name = "space klot"
+	desc = "A powder that, when poured on an open wound, quickly stops the bleeding. Combine with bandages for the best effect."
+	icon = 'icons/obj/medical.dmi'
+	icon_state = "powderbag"
+	heal_brute = 15
+	origin_tech = list(TECH_BIO = 4)
+	var/open = 0
+	var/used = 0
+
+/obj/item/stack/medical/advanced/bruise_pack/spaceklot/attack(mob/living/carbon/M as mob, mob/user as mob)
+	if(..())
+		return 1
+	if(used)
+		return
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		var/obj/item/organ/external/affecting = H.get_organ(user.zone_sel.selecting)
+
+		if(affecting.open == 0)
+			if(affecting.is_bandaged())
+				to_chat(user, "<span class='warning'>The wounds on [M]'s [affecting.name] have already been treated.</span>")
+				return 1
+			else
+				user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
+				user.visible_message("<span class='notice'>\The [user] starts treating [M]'s [affecting.name].</span>", \
+						             "<span class='notice'>You start treating [M]'s [affecting.name].</span>" )
+				if(!do_after(user, 100, act_target = M))
+					return

```

Comment:
Needs a return value.

Reply:
i've gotta be honest, i understand none of this. this is legacy code i just discovered.

Reply:
Tested it, it works just like any other bandage, since it's derived from the same master item. I don't see any need to change it.